lib/cn/node_split.c

```diff
@@ -615,3 +646,31 @@ cn_split_nodes_free(const struct cn_compaction_work *w, struct cn_tree_node *nod
 
     nodev[LEFT] = nodev[RIGHT] = NULL;
 }
+
+void
+cn_split_node_stats_dump(
+    struct cn_compaction_work *w,
+    const struct cn_tree_node *node,
+    const char                *pos)
+{
+    const struct cn_node_stats *ns;
+
+    if (!node)
+        return;
+
+    ns = &node->tn_ns;
+
+    slog_info(
+            SLOG_START("split"),
+            SLOG_FIELD("jobid", "%u", sts_job_id_get(&w->cw_job)),
+            SLOG_FIELD("node", "%s", pos),
+            SLOG_FIELD("cnid", "%lu", w->cw_tree->cnid),
+            SLOG_FIELD("nodeid", "%lu", node->tn_nodeid),
+            SLOG_FIELD("n_kvsets", "%u", cn_ns_kvsets(ns)),
```

Comment:
Should you just remove the `n_` prefix?

Reply:
Better to order somewhat in increasing level of detail like we do in most places, or should do.  At the very least start with cnid followed by node ID, then whatever..  We should actually normalize all messages that print out a lot of the same stuff...

Reply:
Where do you like jobid to be? I'm thinking jobid should be the first followed by cnid and nodeid...

Reply:
I would prefer cnid always be first, but I see other examples of "job, cnid, node id, ...", so just go with that, we can normalize them some rainy day...